README.md

```diff
@@ -14,7 +14,7 @@ You can go to the examples folder, however the usage of the module could be like
 ```hcl
 module "vpc" {
     source  = "terraform-google-modules/network/google"
-    version = "0.6.0"
+    version = "~> 1.0"
```

Comment:
In prior conversation, we've discussed version-pinning to the patch version, e.g. `version = "~> 1.0.0"`. Thoughts on doing that here?

Reply:
See https://github.com/terraform-google-modules/terraform-google-network/pull/53; resolving.